### ISSUE TITLE
USB-C video is not working even though its not connected to NVIDIA dGPU

### ISSUE THREAD

**Issue by adrianjagielak** [adrianjagielak/lenovo_y520_efi#6]
I tried to make it work but I'm not that good at framebuffer patching. If someone is willing to help it will be much easier to solve with more than one head.

**Comment by zty199** [comment]
Try to add boot-arg "agdpmod=vit9696" to see if it works, only if your framebuffer patching is correct......

**Comment by adrianjagielak** [comment]
The biggest problem is framebuffer patching, but now I realize I could have had it at one time but I didn't know about this flag...

**Comment by TomJumel** [comment]
I have a Lenovo y520 with i57300hq and a Nvidia gforce 1500 2gb and without changements my c-video work perfectly with just a c to hdmi cable. Idk why that can't work with your computer.

**Comment by adrianjagielak** [comment]
@Thesoce60 Can you share your EFI so I can take a look?

**Comment by TomJumel** [comment]
I have a dual boot and I use this EFI
[EFI.zip](https://github.com/adrianjagielak/lenovo_y520_efi/files/5797562/EFI.zip)

**Comment by adrianjagielak** [comment]
Wait, are you just using my efi and usb c video is working fine?

**Comment by TomJumel** [comment]
yes sorry for the late response, that's work fine ! But I remember I'm not on Big Sur, I have Catalina installed and I don't know if I update if the c-video will work

**Comment by adrianjagielak** [comment]
You could try to boot the Big Sur pendrive installer and check if c-video works in the installer
